<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Splash</class>
 <widget class="QWidget" name="Splash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font-family: &quot;Roboto&quot;;
	color: #AEAEAE;
}

#Splash {
	background: rgb(62, 204, 156);
}

#centralWidget, #page, #page_2 {
	background: rgb(30, 30, 38);
}

#captionBtnX {
	border-image: url(:/images/xNormal);
}

#captionBtnX:hover {
	border-image: url(:/images/xHover);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="centralWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="caption" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="captionBtnX">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: transparent;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="mainStack">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="content" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="lblLogo">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="resources.qrc">:/images/logoPtb</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="lblLogoText_2">
               <property name="font">
                <font>
                 <family>Roboto</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#3ecc9c;&quot;&gt;Better&lt;/span&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Discord&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>15</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Roboto</family>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;To learn more about privacy, see the BetterDiscord &lt;a href=&quot;https://betterdiscord.net/fwd?id=pp&quot;&gt;&lt;span style=&quot; text-decoration: none; color:#3ecc9c;&quot;&gt;Privacy Policy&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <property name="openExternalLinks">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>15</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="lineWidth">
                <number>1</number>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;By continuing you agree to the BetterDiscord &lt;a href=&quot;https://betterdiscord.net/fwd?id=slt&quot;&gt;&lt;span style=&quot;text-decoration: none; color:#3ecc9c;&quot;&gt;Software License Terms&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <property name="openExternalLinks">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="footer" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="footerSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>405</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnContinue">
               <property name="styleSheet">
                <string notr="true">QPushButton {
	background: transparent;
    color: #fff;
    padding: 7px 10px;
    min-width: 50px;
	background: #17171c;
	border-radius: 5px;
}

QPushButton::hover {
	background: rgb(35, 35, 38);
	background: rgb(62, 204, 156)
}</string>
               </property>
               <property name="text">
                <string>Continue</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="content_2" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="lblLogo_2">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="resources.qrc">:/images/logoPtb</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="lblLogoText_3">
               <property name="font">
                <font>
                 <family>Roboto</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#3ecc9c;&quot;&gt;Better&lt;/span&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Discord&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="systemChecksLayout">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>System Checks</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Pbar" name="pbarSystemChecks">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>8</height>
                  </size>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="remoteChecksLayout">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Remote Resources</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Pbar" name="pbarRemotes">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>8</height>
                  </size>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Pbar</class>
   <extends>QProgressBar</extends>
   <header>pbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>captionBtnX</sender>
   <signal>clicked()</signal>
   <receiver>Splash</receiver>
   <slot>attemptClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnContinue</sender>
   <signal>clicked()</signal>
   <receiver>Splash</receiver>
   <slot>btnContinueClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>attemptClose()</slot>
  <slot>btnContinueClicked()</slot>
 </slots>
</ui>
